Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/GamepadSamplePage.xaml

```diff
@@ -101,79 +101,373 @@ public class GamepadViewModel : ViewModelBase
+																   StrokeThickness="2"
+																   RadiusX="3"
+																   RadiusY="3"
+																   Width="25"
+																   Height="8"
+																   Fill="{Binding ButtonPaddle1, Converter={StaticResource TrueToFilled}, FallbackValue=Transparent}"
+																   Canvas.Left="93"
+																   Canvas.Top="128" />
+														<Rectangle Stroke="#A1A1A1"
+																   StrokeThickness="2"
+																   RadiusX="3"
+																   RadiusY="3"
+																   Width="25"
+																   Height="8"
+																   Fill="{Binding ButtonPaddle2, Converter={StaticResource TrueToFilled}, FallbackValue=Transparent}"
+																   Canvas.Left="93"
+																   Canvas.Top="142" />
+														<Rectangle Stroke="#A1A1A1"
+																   StrokeThickness="2"
+																   RadiusX="3"
+																   RadiusY="3"
+																   Width="25"
+																   Height="8"
+																   Fill="{Binding ButtonPaddle3, Converter={StaticResource TrueToFilled}, FallbackValue=Transparent}"
+																   Canvas.Left="139"
+																   Canvas.Top="128" />
+														<Rectangle Stroke="#A1A1A1"
+																   StrokeThickness="2"
+																   RadiusX="3"
+																   RadiusY="3"
+																   Width="25"
+																   Height="8"
+																   Fill="{Binding ButtonPaddle4, Converter={StaticResource TrueToFilled}, FallbackValue=Transparent}"
+																   Canvas.Left="139"
+																   Canvas.Top="142" />
+
+														<!--Shoulder Buttons-->
+														<Rectangle Stroke="#A1A1A1"
+																   StrokeThickness="2"
+																   RadiusX="3"
+																   RadiusY="3"
+																   Width="25"
+																   Height="8"
+																   Fill="{Binding ButtonLeftShoulder, Converter={StaticResource TrueToFilled}, FallbackValue=Transparent}"
+																   Canvas.Left="77"
+																   Canvas.Top="19" />
+														<Rectangle Stroke="#A1A1A1"
+																   StrokeThickness="2"
+																   RadiusX="3"
+																   RadiusY="3"
+																   Width="25"
+																   Height="8"
+																   Fill="{Binding ButtonRightShoulder, Converter={StaticResource TrueToFilled}, FallbackValue=Transparent}"
+																   Canvas.Left="155"
+																   Canvas.Top="19" />
+
+														<!--Triggers-->
+														<Rectangle Stroke="#A1A1A1"
+																   StrokeThickness="2"
+																   Width="13"
+																   Height="17"
+																   RadiusX="6"
+																   RadiusY="6"
+																   Canvas.Left="89">
+															<Rectangle.Fill>
+																<SolidColorBrush Color="#A1A1A1"
+																				 Opacity="{Binding LeftTrigger}" />
+
+															</Rectangle.Fill>
+														</Rectangle>
+														<Rectangle Stroke="#A1A1A1"
```

Comment:
Perhaps we can create a SolidColorBrush resource for the A1A1A1 color so we'd only need to change it in one place if needed

Reply:
Glad you like it! That is a good idea, will get right on it. I initially wanted to do use a ThemeResource for it, but it did not work with the TrueToFilled Converter, which I use for filling the Shapes.